controllers/UsersController.js

```diff
@@ -83,4 +86,40 @@ export default class UsersController {
       }
     }).catch(next);
   }
+
+  // static verifyUser()
+  /**
+/**
+   * Verify the email sent to the newly registered user
+   * @param {*} req - request object
+   * @param {*} res - response object
+   * @param {*} next - Next function
+   * @returns {token} token - returns JWT token
+   */
+  static async verifySentEmail(req, res) {
```

Comment:
I think a more descriptive name will be verifyEmail. Cause what you are doing is verifying the email Address, not an email sent to them.

Reply:
Thank you @OKiMaureen for this oversight. I Think I get your point. I will act on it.